In [ ]:
import pandas as pd

Dataset de ausentismo laboral

In [ ]:
df = pd.read_excel('/content/Absenteeism_at_work.xls')
df.head()

ID  Reason for absence  Month of absence  Day of the week  Seasons  \
0  11                  26                 7                3        1   
1  36                   0                 7                3        1   
2   3                  23                 7                4        1   
3   7                   7                 7                5        1   
4  11                  23                 7                5        1   

   Transportation expense  Distance from Residence to Work  Service time  Age  \
0                     289                               36            13   33   
1                     118                               13            18   50   
2                     179                               51            18   38   
3                     279                                5            14   39   
4                     289                               36            13   33   

   Work load Average/day   ...  Disciplinary failure  Education  Son  \
0                  239554  ...                     0          1    2   
1                  239554  ...                     1          1    1   
2                  239554  ...                     0          1    0   
3                  239554  ...                     0          1    2   
4                  239554  ...                     0          1    2   

   Social drinker  Social smoker  Pet  Weight  Height  Body mass index  \
0               1              0    1      90     172               30   
1               1              0    0      98     178               31   
2               1              0    0      89     170               31   
3               1              1    0      68     168               24   
4               1              0    1      90     172               30   

   Absenteeism time in hours  
0                          4  
1                          0  
2                          2  
3                          4  
4                          2  

[5 rows x 21 columns]

In [ ]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 740 entries, 0 to 739
Data columns (total 21 columns):
 #   Column                           Non-Null Count  Dtype
---  ------                           --------------  -----
 0   ID                               740 non-null    int64
 1   Reason for absence               740 non-null    int64
 2   Month of absence                 740 non-null    int64
 3   Day of the week                  740 non-null    int64
 4   Seasons                          740 non-null    int64
 5   Transportation expense           740 non-null    int64
 6   Distance from Residence to Work  740 non-null    int64
 7   Service time                     740 non-null    int64
 8   Age                              740 non-null    int64
 9   Work load Average/day            740 non-null    int64
 10  Hit target                       740 non-null    int64
 11  Disciplinary failure             740 non-null    int64
 12  Education                        740 non-null    i

1) Fuente de datos: [Repositorio de machine learning de UCI](https://archive.ics.uci.edu/ml/datasets/Absenteeism+at+work)

2) Breve descripción de datos: Dataset sobre ausentismo laboral y las distintas variables que lo explican. Datos utilizados en investigación académica en la Universidade Nove de Julho - Programa de Posgrado en Informática y Gestión del Conocimiento

3) ¿Cuál es el objetivo?: Conocer variables que explican el ausentismo laboral con estos datos especificos. Modelo que podría servir como referencia a tomar en cuenta y comparar con otros datos relacionados a la misma materia.

4) ¿Este es un problema de clasificación o regresión?: Problema de clasificación

5) ¿Cuántas características hay?: 21 caracteristicas (columnas)

6) ¿Cuántas filas de datos hay?: 740 filas

7) ¿Qué retos, si es que hay alguno, prevén en la limpieza, exploración o modelización con este conjunto de datos?: Elegir de buena forma las caracteristicas que realmente impactan en el resultado final, cuales son relaventes en el análisis y cuales no. Esto puede implicar eliminar incluso algunas columnas.

Dataset de intención de compra online

In [ ]:
df2 = pd.read_csv('/content/online_shoppers_intention.csv')
df2.head()

Administrative  Administrative_Duration  Informational  \
0               0                      0.0              0   
1               0                      0.0              0   
2               0                      0.0              0   
3               0                      0.0              0   
4               0                      0.0              0   

   Informational_Duration  ProductRelated  ProductRelated_Duration  \
0                     0.0               1                 0.000000   
1                     0.0               2                64.000000   
2                     0.0               1                 0.000000   
3                     0.0               2                 2.666667   
4                     0.0              10               627.500000   

   BounceRates  ExitRates  PageValues  SpecialDay Month  OperatingSystems  \
0         0.20       0.20         0.0         0.0   Feb                 1   
1         0.00       0.10         0.0         0.0   Feb                 2   
2         0.20       0.20         0.0         0.0   Feb                 4   
3         0.05       0.14         0.0         0.0   Feb                 3   
4         0.02       0.05         0.0         0.0   Feb                 3   

   Browser  Region  TrafficType        VisitorType  Weekend  Revenue  
0        1       1            1  Returning_Visitor    False    False  
1        2       1            2  Returning_Visitor    False    False  
2        1       9            3  Returning_Visitor    False    False  
3        2       2            4  Returning_Visitor    False    False  
4        3       1            4  Returning_Visitor     True    False

In [ ]:
df2.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 12330 entries, 0 to 12329
Data columns (total 18 columns):
 #   Column                   Non-Null Count  Dtype  
---  ------                   --------------  -----  
 0   Administrative           12330 non-null  int64  
 1   Administrative_Duration  12330 non-null  float64
 2   Informational            12330 non-null  int64  
 3   Informational_Duration   12330 non-null  float64
 4   ProductRelated           12330 non-null  int64  
 5   ProductRelated_Duration  12330 non-null  float64
 6   BounceRates              12330 non-null  float64
 7   ExitRates                12330 non-null  float64
 8   PageValues               12330 non-null  float64
 9   SpecialDay               12330 non-null  float64
 10  Month                    12330 non-null  object 
 11  OperatingSystems         12330 non-null  int64  
 12  Browser                  12330 non-null  int64  
 13  Region                   12330 non-null  int64  
 14  TrafficType           

1) Fuente de datos: [Repositorio de machine learning UCI](https://archive.ics.uci.edu/ml/datasets/Online+Shoppers+Purchasing+Intention+Dataset)

2) Breve descripción de datos: Dataset enfocado en intención de compra de usuario de página web, tomando en cuenta la exploración del sitio antes de concretar una compra.

3) ¿Cuál es el objetivo?: Llegar a conclusiones sobre patrones de conducta que tienen los compradores de la página.

4) ¿Este es un problema de clasificación o regresión?: Problema de clasificación

5) ¿Cuántas características hay?: 18 caracteristicas (columnas)

6) ¿Cuántas filas de datos hay?: 12.330 filas

7) ¿Qué retos, si es que hay alguno, prevén en la limpieza, exploración o modelización con este conjunto de datos?: Será interesante llegar a conclusiones y trabajar un conjunto de datos que cuenta con variados tipos de datos.